python-sdk/tests/sql/test_table.py

```diff
@@ -151,3 +152,74 @@ def test_if_table_object_can_be_pickled():
     """Verify if we can pickle Table object"""
     table = Table()
     assert pickle.loads(pickle.dumps(table)) == table
+
```

Comment:
We should have a unit or integration test to check the if extractor is called when operator is called

Reply:
For example: https://github.com/astronomer/astro-sdk/blob/main/python-sdk/tests/extractors/test_extractor.py#L59